testing/src/cypress/e2e/02-institution/01-login-institution-page.cy.ts

```diff
@@ -1,22 +1,34 @@
 import DashboardInstitutionObject from "../../page-objects/Institution-objects/DashboardInstitutionObject";
 import InstitutionCustomCommand from "../../custom-command/institution/InstitutionCustomCommand";
+import LoginInstitutionObject from "../../page-objects/Institution-objects/LoginInstitutionObject";
 
-describe("Login Page", () => {
-  const dashboardInstitutionObject = new DashboardInstitutionObject();
-  const institutionCustomCommand = new InstitutionCustomCommand();
+const dashboardObject = new DashboardInstitutionObject();
+const iCC = new InstitutionCustomCommand();
+const loginObject = new LoginInstitutionObject();
 
-  const url = Cypress.env("institutionURL");
+const LOGIN_URL = Cypress.env("TEST").BASE_URL + "/institution/login";
+const UNAME = Cypress.env("TEST").UNAME_1;
+const PASS = Cypress.env("TEST").PASS_1;
 
-  before(() => {
-    cy.visit(url);
+describe("Login Page", () => {
+  beforeEach(() => {
+    cy.visit(LOGIN_URL);
   });
 
-  it("Verify login case in institution.", () => {
-    institutionCustomCommand.loginInstitution();
+  it("Verify invalid uname/password field validation error", () => {
+    iCC.loginWithCredentials("invalid", "invalid");
+    loginObject
+      .errorMessage()
+      .should("have.text", "The user ID or password you entered is incorrect");
   });
 
-  it("Verify that clicking on the Log Off then it must be log out.", () => {
-    dashboardInstitutionObject.profileIconButton().click();
-    dashboardInstitutionObject.logOffButton().click();
+  it("Verify login and logout successfully logout", () => {
```

Comment:
What about having login and logout as separate tests.

Reply:
Sure, we can separate them.

Reply:
Addressed